susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -33,11 +33,11 @@ mgr_server_localhost_alias_absent:
 {% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/' ~ os_base ~ '/' ~ grains['osmajorrelease'] ~ '/bootstrap/' %}
 {% endif %}
 {%- elif grains['os_family'] == 'Debian' %}
-{%- set osrelease = grains['osrelease'].split('.') %}
+#{%- set osrelease = grains['osrelease'].split('.') %}
```

Comment:
@gnuninu why is this commented?

Looks to me that without this, Ubuntu won't work, as it uses `osrelease[0]` and `osrelease[1]`

Or am I missing anything?

Reply:
> @gnuninu why is this commented?
that change was probably inherited by the first tests i ran to fix the URL for Debian, indeed it's necessary for Ubuntu so i am removing the comment.
> 
> Looks to me that without this, Ubuntu won't work, as it uses `osrelease[0]` and `osrelease[1]`
Correct because it sets `osrelease` with the python string method `split('.')`
> 
yes 
> Or am I missing anything?